var/spack/repos/builtin/packages/opensubdiv/package.py

```diff
@@ -0,0 +1,85 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+
+
+class Opensubdiv(CMakePackage, CudaPackage):
+    """OpenSubdiv is a set of open source libraries that
+    implement high performance subdivision surface (subdiv)
+    evaluation on massively parallel CPU and GPU architectures.
+    This code path is optimized for drawing deforming surfaces
+    with static topology at interactive framerates."""
+
+    homepage = "http://graphics.pixar.com/opensubdiv/docs/intro.html"
+    url      = "https://github.com/PixarAnimationStudios/OpenSubdiv/archive/v3_4_0.tar.gz"
+    git      = "https://github.com/PixarAnimationStudios/OpenSubdiv"
+
+    version('dev', branch='dev')
+    version('3_4_0',     sha256='d932b292f83371c7518960b2135c7a5b931efb43cdd8720e0b27268a698973e4')
+
+    variant('tbb', default=False, description='Builds with Intel TBB support')
+    variant('openmp', default=False, description='Builds with OpenMP support')
+    variant('doc', default=False, description='Builds documentation. Requires python@@2.6')
```

Comment:
Why the double @?

Reply:
`@` needs to be escaped with `@@` else `==> Error: Incomplete color format: '@ in python@2.6` for command `spack info opensubdiv`

Reply:
Hmm, that's probably a bug. Want to open up a separate issue for this? In the meantime, you can just change it to "Python 2", or not mention the dependency at all.

Reply:
fixed